autoclasstoc/sections.py

```diff
@@ -131,6 +131,8 @@ def exclude_if_match(exclude_pattern, name):
         """
         if exclude_pattern:
             return any(re.search(p, name) for p in exclude_pattern)
+        else:
+            return False
```

Comment:
```suggestion
        return exclude_pattern and any(re.search(p, name) for p in exclude_pattern)
```

Reply:
Applied suggestions and changed code tests accordingly